ATTACK/Assets/Scripts/CharacterCommon.cs

```diff
@@ -20,4 +26,19 @@ public void SetTeam(Team team)
     {
         this.team = team;
     }
+
+    public Team GetTeam()
+    {
+        return this.team;
+    }
+
+    private void Attack()
+    {
+        Transform targetPoint = gameManager.GetRandomTarget(team);

```

Comment:
Rotate the character object on attack so that it faces the enemy.
I built the attack-system so that a projectile travels in the direction of the character.